python/semantic_kernel/contents/image_content.py

```diff
@@ -45,18 +47,23 @@ class ImageContent(BinaryContent):
 
     Raises:
         ValidationError: If neither uri or data is provided.
-
     """
 
     content_type: Literal[ContentTypes.IMAGE_CONTENT] = Field(IMAGE_CONTENT_TAG, init=False)  # type: ignore
     tag: ClassVar[str] = IMAGE_CONTENT_TAG
 
     @classmethod
+    @deprecated("The `from_image_path` method is deprecated; use `from_image_file` instead.", category=None)
     def from_image_path(cls: type[_T], image_path: str) -> _T:
         """Create an instance from an image file."""
-        mime_type = mimetypes.guess_type(image_path)[0]
-        with open(image_path, "rb") as image_file:
-            return cls(data=image_file.read(), data_format="base64", mime_type=mime_type, uri=image_path)
+        return cls.from_image_file(image_path)
+
+    @classmethod
+    def from_image_file(cls: type[_T], path: str) -> _T:
```

Comment:
Because in the comment for this class, it says `from_image_file`. And I think `from_image_file` also makes more sense than `from_image_path`. The new audio content also follows the same pattern: `from_audio_file()`.